### DISCUSSION TITLE
most used languages doesnt work on my profile

### DISCUSSION THREAD

**Discussion by nabilramy** [anuraghazra/github-readme-stats#1745] [Q&A] [answered]
hi there, why my used languages states doesn't work !!

**Comment by rickstaa** [comment, marked as answer]
@nabilramy Thanks for your question. I see you are indeed using the correct syntax:

```markdown
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=nabilramy&layout=compact)](https://github.com/anuraghazra/github-readme-stats)
````

I, however, noticed that you don't have any public repositories of your own other than the readme. As also explained in https://github.com/anuraghazra/github-readme-stats/discussions/1255 we currently don't count contributions in forks. They will only be counted if they are merged into the main repository. Apart from that, we also don't count contributions to private repositories. You have to deploy your own Vercel instance to make those show up (See https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance).

**Reply by rickstaa** [reply]
@nabilramy Did that answer your question?

**Reply by arnab4477** [reply]
Not OP but helpful answer.

**Reply by rickstaa** [reply]
> Not OP but helpful answer.

Can you elaborate on how you still need help? We directly use the results from GraphQL, so you can check the results returned by GitHub directly using the [GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer):

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    # fetch only owner repos & not forks
    repositories(ownerAffiliations: OWNER, isFork: false, first: 100) {
      nodes {
        name
        languages(first: 10, orderBy: {field: SIZE, direction: DESC}) {
          edges {
            size
            node {
              color
              name
            }
          }
        }
      }
    }
  }
}
```

```graphql
{"login": "arnab4477"}
```

**Reply by arnab4477** [reply]
I don't need help. I just said I am not the OP (original poster, here the one who originally asked the question) but still found it helpful

**Comment by anselal** [comment]
I am facing the same issue. The link I use is this https://github-readme-stats.vercel.app/api/top-langs/?username=anselal&hide=php,c,vim%20script,objective-c,actionscript,roff,html,javascript,css,java,&langs_count=3&theme=nord
which displays the stats even from a browser where I am not logged in.
But in my profile the image does not show up https://github.com/anselal/anselal

The exact code I am using is 

```
<a href="https://github.com/anselal/anselal">
  <img align="center" src="https://github-readme-stats.vercel.app/api/top-langs/?username=anselal&hide=php,c,vim script,objective-c,actionscript,roff,html,javascript,css,java,&langs_count=3&theme=nord" />
</a>
```

<a href="https://github.com/anselal/anselal">
  <img align="center" src="https://github-readme-stats.vercel.app/api/top-langs/?username=anselal&hide=php,c,vim script,objective-c,actionscript,roff,html,javascript,css,java,&langs_count=3&theme=nord" />
</a>

As you can see it does not show up here either, but all other images do.
Do I have an error in my html ?

**Reply by rickstaa** [reply]
@anselal Thanks for your question. The right syntax should be:

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anselal)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anselal)](https://github.com/anuraghazra/github-readme-stats)
````

<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=anselal" />

```md
<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=anselal" />
```

**Reply by anselal** [reply]
thnx. my format works just fine with the link you provided.
I found that a parameter was causing the image to not render, particularly the "vim script" part in "hide="

So `hide=vim script` breaks the image. But how can I hide it ? As you see in your image it is "Vim Script". How do I exclude it ?

**Reply by rickstaa** [reply]
@anselal That is expected since URLs can not handle spaces (see https://github.com/anuraghazra/github-readme-stats#language-card-exclusive-options). You should use `hide=vim%20script` instead.

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anselal&hide=vim%20script)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anselalhide=vim%20script)](https://github.com/anuraghazra/github-readme-stats)
````

**Comment by messiisgreat** [comment]
Thank you!
